### ISSUE TITLE
Storing absorbed fixed effects

### ISSUE THREAD

**Issue by sdarougheh** [ksecology/FixedEffectModel#1]
Can the absorbed fixed effects be recovered/stored, as in the STATA version of reghdfe?

**Comment by schouxy53** [comment]
yes, we will be uploading a new version soon with a bunch of new functionalities, stay tuned.

**Comment by schouxy53** [comment]
Hi, we just released version 0.0.3, and you can use getfe(result) to recover fixed effect.

**Comment by yelinhu** [comment]
Is it possible to have clustering but no fixed effects with this package? I tried one with no fixed effects (category_input) and one cluster input but the regression seems to think I want the cluster as a fixed effect.

**Comment by schouxy53** [comment]
Yes you can.

_consist_input = ['x_1','x_2']
out_input = ['y']
category_input = []
cluster_input = ['id','time']
result1 = ols_high_d_category(data_df,
                              consist_input,
                              out_input,
                              category_input,
                              cluster_input)
result1.summary()_

Code above allow you cluster on ['id','time'] level but no fixed effect.


> Is it possible to have clustering but no fixed effects with this package? I tried one with no fixed effects (category_input) and one cluster input but the regression seems to think I want the cluster as a fixed effect.

**Comment by yelinhu** [comment]
Hi! Is there a way to store the summary results of the OLS into a csv/excel file? Thanks!

**Comment by schouxy53** [comment]
Hi, currently the summary table cannot be saved into the excel file, but you can extract the information from the result class: “result.params” is beta, “result.bse” is standard error, etc.
